src/components/Content/Pursuance/PursuancePage.js

```diff
@@ -9,32 +9,76 @@ import DiscussView from './views/DiscussView';
 import ParticipantsView from './views/ParticipantsView';
 import RightPanel from '../RightPanel/RightPanel';
 import './PursuancePage.css';
+import ReactModal from 'react-modal';
 
 class PursuancePage extends Component {
-
+ constructor(){
+  super();
+  this.state={
+    modalIsOpen : false
+   }
+ }
   componentWillMount() {
     let { setCurrentPursuance, match, currentPursuanceId } = this.props;
     currentPursuanceId = Number(match.params.pursuanceId) || currentPursuanceId;
     setCurrentPursuance(currentPursuanceId);
+    ReactModal.setAppElement('body');
+  }
+  componentDidMount(){
+   this.openModal();
+  }
+  openModal = () => {
+   this.setState({modalIsOpen : true});
+  }
+  closeModal = () => {
+   this.setState({modalIsOpen: false});
+  }
+  handleKeyDown = (event) => {
+   if(event.key === "?"){
+    this.setState({modalIsOpen : true});
+   }
   }
-
   render() {
     return (
-      <Router>
-        <div id="pursuance-page" className="content-ctn">
-          <nav id="pursuance-nav">
-            <PursuanceMenu />
-          </nav>
-          <article>
-            <Switch>
-              <Route exact path="/pursuance/:pursuanceId" component={TaskListView} />
-              <Route exact path="/pursuance/:pursuanceId/tasks" component={TaskListView} />
-              <Route exact path="/pursuance/:pursuanceId/calendar" component={CalendarView} />
-              <Route exact path="/pursuance/:pursuanceId/discuss/task/:taskGid" component={DiscussView} />
-              <Route exact path="/pursuance/:pursuanceId/participants" component={ParticipantsView} />
-            </Switch>
-            <RightPanel />
-          </article>
+    <Router>
+        <div id="pursuance-page" className="content-ctn" onKeyDown={this.handleKeyDown} tabIndex="-1">
+        <ReactModal className="Modal" shouldCloseOnEsc={true} shouldFocusAfterRender={true} isOpen={this.state.modalIsOpen} onRequestClose={this.closeModal} closeTimeoutMS={350} style={{overlay:{background:"none"}}}>
+          <span className='close' onClick={this.closeModal}>&times;</span>
+          <h3>Keyboard Shortcuts</h3>
+          <hr style={{border:"1.5px solid grey"}}/>
+            <div className="columns">
+              <div className="column one-half">
+                <table className="keyboard-mappings">
+                  <tbody>
+          j         <tr>
+                      <td className="keys">
+                        <kbd>?</kbd>
+                      </td>
+                      <td>&nbsp; Open shortcut window</td>
+                    </tr>
+                    <tr>
+                      <td className="keys">
+                        <kbd>Everything Else</kbd>
+                      </td>
+                      <td> &nbsp; Do Nothing</td>
+                    </tr>
+                  </tbody> </table>
+              </div>
+            </div>
+        </ReactModal>
```

Comment:
@greg-han Turn this into its own component and then use it here just like `PursuanceMenu` or the like is used -- like this: `<PursuanceMenu />` (rather than putting all the code in-line).

Reply:
@greg-han Sorry for all the changes, but the React way of doing things is very different from the vanilla JS way.

Reply:
No problem, I appreciate the feedback!
I'll get on that.